src/libs/Logger.js

```diff
@@ -46,19 +43,43 @@ module.exports = class Logger {
     return ErrorNoStack;
   }
 
-  buildLog (logLevelindex, event, data = null, meta = null) {
-    if (data instanceof Error) {
-      let error = data;
+  serializeError (logLevelindex, logData) {
+    let serialized;
+    const isError = logData instanceof Error;
+    let baseError;
+
+    if (isError) {
+      baseError = logData;
+    } else if ((logData || {}).error instanceof Error) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing "else" clause. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AElseIfWithoutElse)